templates/admin/lbf/edit_lbf/partials/modify_group.html.twig

```diff
@@ -0,0 +1,37 @@
+<!-- template DIV that appears when user chooses to add a new group -->
+<div class="modal" id="{{ id }}group" tabindex="-1" aria-labelledby="{{ id }}GroupModal" aria-hidden="false">
+    <div class="modal-dialog modal-lg modal-dialog-scrollable">
+        <div class="modal-content">
+            <div class="modal-header">
+                {% set modalTitle = (id == "rename") ? "Rename Group" : "Add New Group" %}
+                {% set btnSaveValue = (id == "rename") ? "Rename Group" : "Save New Group" %}
+                <h5 class="modal-title" id="{{ id }}GroupModal">{{ modalTitle|xlt }}</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="{{ 'Close'|xla }}">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                {% if id == "rename" %}
+                    <input type="hidden" name="renameoldgroupname" id="renameoldgroupname" value="" />
+                {% endif %}
+                <div class="form-group">
+                    <label for="{{ id }}groupname" class="form-label">{{ "Group Name"|xlt }}</label>
+                    <input type="text" size="20" maxlength="30" name="{{ id }}groupname" id="{{ id }}groupname" class="form-control">
+                </div>
+                <div class="form-group">
+                    <label for="{{ id }}groupparent" class="form-label">{{ "Parent"|xlt }}</label>
+                    <select name="{{ id }}groupparent" class="select2 form-control" id="{{ id }}group_parent">
+                        <option value="">{{ "None{{group}}"|xlt }}</option>
+                        {% for group in groups %}
+                            <option value="{{ group.id|attr }}" {{ group.default }}>{{ group.title|text }}</option>
```

Comment:
might as well pipe group.default through attr (guessing this is just selected or something, but won't hurt)

Reply:
Is there really any benefit to running it through the attribute function though?

Reply:
if somehow, some way somebody modifies code in future for group.default to be user defined and get javascript in there, then yes (I honestly don't see how it would be possible, but since these code elements are now further removed 
 from each other maybe a future coding mistake or something).
As above discussed, though, no harm done, easy to do, and then never worry about again.